onnxruntime/test/util/default_providers.cc

```diff
@@ -39,6 +39,9 @@ std::unique_ptr<IExecutionProvider> DefaultTensorrtExecutionProvider() {
       0,
       0,
       0,
+      0,
+      0,
+      0,
```

Comment:
We shouldn't increase arguments for OrtTensorRTProviderOptions since it is a legacy struct and is deprecated, please remove these three arguments. We only add new fields to OrtTensorRTProviderOptionsV2. I think it's the reason that pipeplines failed.

Reply:
Done